force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -102,16 +111,18 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer countVowels(String str) {
         // Initialize the result Integer
-
-        /*
-        for () {
+        Integer result = 0;
+        List<String> vowels = new List<String>{'a', 'e', 'i', 'o', 'u'};
+        for (String chr: str.split('')) {
             // get individual characters from the string - hint use substring or split
             // check if the character is a vowel
-            // if it is a vowel, increment the result Integer
+            if (vowels.contains(chr.toLowerCase())) {
+                // if it is a vowel, increment the result Integer
+                result++;
+            }
```

Comment:
So when you are checking for membership like is this letter in the set of vowels use a set rather than a list. Lists take linear time for checking member while a set is constant time.